apps/auth/auth_cookie.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from datetime import timedelta
+from flask import make_response, current_app as app, request, g
+from superdesk import Blueprint
+from superdesk.utc import utcnow
+
+auth_cookie_bp = Blueprint("auth_cookie_bp", __name__)
+
+
+@auth_cookie_bp.route("/auth_cookie", methods=["POST", "OPTIONS"])
+def get_auth_test_post():
```

Comment:
imo this method can use the `blueprint_auth` decorator as well
and we could use `DELETE` method to delete cookie